.github/workflows/release.yml

```diff
@@ -19,15 +19,11 @@ jobs:
         uses: actions/setup-go@v2
         with:
           go-version: 1.16
-      -
-        name: Find version
-        id: version
-        run: echo ::set-output name=tag::${GITHUB_REF#refs/tags/v}
       -
         name: Run GoReleaser
         uses: goreleaser/goreleaser-action@v2
         with:
-          version: ${{ steps.version.outputs.tag }}
+          version: latest
```

Comment:
Does `latest` do something special or are you setting it to 'latest' for now? More of a question so I understand how versioning will work.

Reply:
This is a nuance with the github action "goreleaser". I originally thought version meant `version` of the library we were releasing. Instead, `version` refers to the goreleaser version to use.